src/annotations.rst

```diff
@@ -267,11 +267,22 @@ specify which method(s) a test method wants to test:
         $this->assertSame(0, $this->ba->getBalance());
     }
 
-If provided, only the code coverage information for the specified
-method(s) will be considered.
+If provided, this effectively filters the code coverage report
+to include executed code from the referenced code parts only.
+
+This annotation can be added to the PHPDoc of the test class or the individual
```

Comment:
Please use "docblock" instead of "PHPDoc". Thanks!

Reply:
Done and repushed.